src/redispatch.js

```diff
@@ -70,7 +68,7 @@ const handlePlaying = (player, event) => {
 const handleEnded = (player, event) => {
   if (player.ads.isInAdMode()) {
 
-    // The true ended event fired by plugin.js either after the postroll
+    // The true ended event fired either after the postroll
     // or because there was no postroll.
     if (player.ads.isContentResuming()) {
```

Comment:
is this the sign that this is the true `ended` event?

Reply:
Yup